test/embedding.jl

```diff
@@ -0,0 +1,12 @@
+# test that the embedding example runs without error
+let
+    if is_windows()
+        embedding_bin = joinpath(JULIA_HOME,Base.LIBEXECDIR,"embedding.exe")
+    else
+        embedding_bin = joinpath(JULIA_HOME,Base.LIBEXECDIR,"embedding")
+    end
+    lines = readlines(pipeline(`$(embedding_bin)`,
+                               stderr=DevNull))
+    @test length(lines) == (is_windows() ? 6 : 5)
```

Comment:
Ah, it doesn't seem to be able to find `my_c_sqrt` on linux. I think `jl_load_dynamic_library_(NULL)` was expected to return the process handle after all.